webpack/webpack.dev.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
               sourceMap: true,
               localsConvention: 'camelCase',
               modules: {
-                localIdentName: '[local]___[hash:base64:5]'
+                localIdentName: '[local]'
```

Comment:
why do you remove hash_base?

Reply:
because my js was sticking the class names I wanted but the compiled CSS came with random strings at the end of class names, therefore, my CSS didn't work and this is the fix I found.